manifests/server/read_database_ini.pp

```diff
@@ -42,12 +45,16 @@
       mode   => '0600';
     }
 
+    $ini_setting_require = str2bool($database_validate) ? {
+      false => undef,
+      default  => Class['puppetdb::server::validate_read_db'],
+    }
     # Set the defaults
     Ini_setting {
       path    => "${confdir}/read_database.ini",
       ensure  => present,
       section => 'read-database',
-      require => Class['puppetdb::server::validate_db'],
```

Comment:
This is a change of behaviour. By all means switch to validate_read_db as necessary.

My only requirement is that I can disable validation so we can build AWS AMIs without requiring a postgresql connection.